src/invoke.cpp

```diff
@@ -4007,18 +4010,19 @@ void lhs_threaded(lk::invoke_t &cxt, wxString &workdir, int &sv, int &num_sample
 	if (wxFileExists(workdir + "/LHS.ERR"))
 	{
 		err_msg = "LHS error.  There could be a problem with the input setup.";
-		FILE *ferr = fopen(wxString(workdir + "/LHS.ERR").c_str(), "r");
-		if (ferr)
+		std::ifstream ferr;
```

Comment:
Changes would have to be tested with LHS sampling in PVRPM code for any issue with file access

Reply:
There really should be a testsuite!

Reply:
We agree! We're still working on catching up on our test coverage for our codebases- it's easier to add tests to sections we're actively working on, but we hope to be able to get more complete coverage through the upcoming years! If you happen to get motivated to add some tests, we'd be thrilled! =)

Reply:
@bje- Could you please build SAM from this branch and run a stochastic simulation? (Right below the "Simulate" button on lower left-hand side of SAM case page).

Reply:
Just did this and the stochastic simulation worked! Can we merge this?

Reply:
@sjanzou thoughts?